configure.ac

```diff
@@ -327,14 +362,23 @@ AC_OUTPUT
 
 AM_COND_IF([ENABLE_TCTI_DEVICE], [],
            [AM_COND_IF([ENABLE_TCTI_MSSIM], [],
-                       [AC_MSG_WARN("No build-in TCTI module enabled")])])
+                       [AM_COND_IF([ENABLE_TCTI_FUZZING], [],
+                                   [AC_MSG_WARN("No build-in TCTI module enabled")])])])
+
+AM_COND_IF([ENABLE_TCTI_FUZZING], [
+            AM_COND_IF([ENABLE_TCTI_DEVICE],
+                       AC_MSG_ERROR("Fuzzing TCTI is meant to be built as the only TCTI"), [])
```

Comment:
@tstruk I made those three changes, remove 2nd commit, change variable names, and add guards to configure.ac to make sure fuzzing TCTI isn't used with either of the others.